frontend/src/components/elements/Balloons/Balloons.test.tsx

```diff
@@ -36,12 +42,23 @@ describe("Balloons element", () => {
     const wrapper = shallow(<Balloons {...props} />)
 
     expect(wrapper).toBeDefined()
-    expect(wrapper.find(".balloons img").length).toBe(15)
+    expect(wrapper.find(".balloons img").length).toBe(NUM_BALLOONS)
 
     wrapper.find(".balloons img").forEach(node => {
       expect(node.prop("src")).toBeTruthy()
       expect(node.prop("style")).toHaveProperty("left")
       expect(node.prop("style")).toHaveProperty("animationDelay")
     })
   })
+
+  it("should render one time per session", async () => {
+    jest.setTimeout(30000)
+
+    const props = getProps()
+    const wrapper = shallow(<Balloons {...props} />)
+
+    await timeout(MAX_ANIMATION_DURATION_MS + DELAY_MAX_MS + 100)
+
+    expect(wrapper.html()).toBeNull()
+  })
```

Comment:
Does `timeout()` actually start a timer in this test? Or is it mocked somehow?

If _not_ mocked, this could be flaky. Plus it slows down our testing a bit.

In case it helps, it looks like Jest has mock timers: https://jestjs.io/docs/en/timer-mocks.html

Reply:
Done